### ISSUE TITLE
Support rubocop-graphql

### ISSUE THREAD

**Issue by HunterMeyer** [codeclimate/codeclimate-rubocop#275]
We use GraphQL in our Rails app and have installed the [rubocop-graphql](https://github.com/DmitryTsepelev/rubocop-graphql) gem to enforce a style guide.

This installation, however, breaks CodeClimate because it can't find the library.

```
/usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:117:in `require': cannot load such file -- rubocop-graphql (LoadError)
```

We have this in our Gemfile:
```ruby
group :development do
  gem 'rubocop-graphql', require: false
end 
```

**Comment by rubendinho** [comment]
Hi CodeClimate team - any updates on this? 

There's a duplicate issue here fwiw: https://github.com/codeclimate/codeclimate-rubocop/issues/275

**Comment by itay-grudev** [comment]
This is still a problem.

**Comment by itay-grudev** [comment]
@noelia-lencina

**Comment by itay-grudev** [comment]
Duplicate of: #268

**Comment by HunterMeyer** [comment]
Closing.

Merged in #278.
